Tests/NostrSDKTests/Events/RelayListMetadataEventTests.swift

```diff
@@ -0,0 +1,140 @@
+        XCTAssertEqual(relayMetadataList[2].relayURL.absoluteString, relayURL3.absoluteString)
+        XCTAssertEqual(relayMetadataList[2].marker, .readAndWrite)
+        XCTAssertEqual(relayMetadataList[2].tag, tag3)
+        XCTAssertEqual(relayMetadataList[3].relayURL.absoluteString, relayURL4.absoluteString)
+        XCTAssertEqual(relayMetadataList[3].marker, .readAndWrite)
+        XCTAssertEqual(relayMetadataList[3].tag, tag4)
+        XCTAssertEqual(relayMetadataList[4].relayURL.absoluteString, relayURL5.absoluteString)
+        XCTAssertEqual(relayMetadataList[4].marker, .write)
+        XCTAssertEqual(relayMetadataList[4].tag, tag5)
+        XCTAssertEqual(relayMetadataList[5].relayURL.absoluteString, relayURL6.absoluteString)
+        XCTAssertEqual(relayMetadataList[5].marker, .read)
+        XCTAssertEqual(relayMetadataList[5].tag, tag6)
+
+        try verifyEvent(event)
+    }
+
+    func testDecodeRelayListMetadata() throws {
+        let event: RelayListMetadataEvent = try decodeFixture(filename: "relay_list_metadata")
+        XCTAssertEqual(event.id, "68962e8499c2067306b2daaa8811b95f38d5e7b8954976d15a8419751a96757a")
+        XCTAssertEqual(event.pubkey, "cb9f20cbd8616dcb79ce1dbdcec702b9b1549e678225ce035b31db4d820f4418")
+        XCTAssertEqual(event.createdAt, 1720822990)
+        XCTAssertEqual(event.kind, .relayListMetadata)
+        XCTAssertEqual(event.content, "")
+        XCTAssertEqual(event.signature, "0b40f4f5de3f6cb2223a7961d7b2a3c8f3b5944a275dc27ba3cea765f6be127fb311f10218e25001bee0e30ac6a2ed7dfa45d8c77e3973ce599eaa0bd1e2423b")
+
+        let publicKey = try XCTUnwrap(PublicKey(hex: event.pubkey))
+        let expectedEventCoordinates = try XCTUnwrap(EventCoordinates(kind: .relayListMetadata, pubkey: publicKey))
+        XCTAssertEqual(event.replaceableEventCoordinates(), expectedEventCoordinates)
+
+        let tag1 = Tag(name: "r", value: "wss://relay.momostr.pink/")
+        let tag2 = Tag(name: "r", value: "wss://relay.primal.net/", otherParameters: ["read"])
+        let tag3 = Tag(name: "r", value: "wss://relay.nostr.band/", otherParameters: ["read"])
+
+        XCTAssertEqual(event.tags, [tag1, tag2, tag3])
+
+        XCTAssertEqual(event.relayMetadataList.count, 3)
+        XCTAssertEqual(event.relayMetadataList[0].relayURL.absoluteString, "wss://relay.momostr.pink/")
+        XCTAssertEqual(event.relayMetadataList[0].marker, .readAndWrite)
+        XCTAssertEqual(event.relayMetadataList[0].tag, tag1)
+        XCTAssertEqual(event.relayMetadataList[1].relayURL.absoluteString, "wss://relay.primal.net/")
+        XCTAssertEqual(event.relayMetadataList[1].marker, .read)
+        XCTAssertEqual(event.relayMetadataList[1].tag, tag2)
+        XCTAssertEqual(event.relayMetadataList[2].relayURL.absoluteString, "wss://relay.nostr.band/")
+        XCTAssertEqual(event.relayMetadataList[2].marker, .read)
+        XCTAssertEqual(event.relayMetadataList[2].tag, tag3)
+    }
+
```

Comment:
Would it be good to test decoding malformed data? Maybe a relay puts an invalid url in. Or maybe they put something other than "read" or "write" where the marker should be.